src/components/strategies/create/BuySellBlock/InputLimit.tsx

```diff
@@ -1,10 +1,12 @@
-import { useFiatCurrency } from 'hooks/useFiatCurrency';
-import { Token } from 'libs/tokens';
 import { ChangeEvent, FC, FocusEvent } from 'react';
 import { carbonEvents } from 'services/events';
+import { useFiatCurrency } from 'hooks/useFiatCurrency';
+import { Token } from 'libs/tokens';
 import { sanitizeNumberInput } from 'utils/helpers';
 import { decimalNumberValidationRegex } from 'utils/inputsValidations';
 import { ReactComponent as IconWarning } from 'assets/icons/warning.svg';
+import { MarketPriceIndication } from 'components/strategies/marketPriceIndication';
+import { MarketPricePercentage } from 'hooks/useMarketIndication';
```

Comment:
I dont think the global hooks folder is the correct place for this file. Do we plan on using this outside of create strategy scope?

Reply:
you right, moving it to strategy scope. we are using it for both edit & create strategy